src/agent/codecoverageconstants.ts

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// Enable Jacoco Code Coverage for Gradle builds using this props
+export function jacocoGradleMultiModuleEnable(excludeFilter: string, includeFilter: string, classFileDirectory: string, reportDir: string) {
```

Comment:
also functions in constants file doesnt make sense. Lets keep it codecoverageUtilties